src/Type.cpp

```diff
@@ -247,6 +254,25 @@ ChooseRandomTypeFilter::get_type()
 
 ///////////////////////////////////////////////////////////////////////////////
 
+// Helper functions
+
+///////////////////////////////////////////////////////////////////////////////
+
+static bool checkImplicitNontrivialAssignOps(vector<const Type*> fields)
+{
+    if (!CGOptions::lang_cpp()) return false;
+    for (size_t i = 0; i < fields.size(); ++i) {
+        const Type* field = fields[i];
+		if (field->has_implicit_nontrivial_assign_ops()){
```

Comment:
Bad indentation? I know we have quite a few indentation issues in the existing code base...